### ISSUE TITLE
Custom word lists

### ISSUE THREAD

**Issue by julianLapenna12** [johnsamuelwrites/inclusive#1]
Does this library support using custom word lists, such as that offered here: https://inclusivenaming.org/?

**Comment by johnsamuelwrites** [comment]
@julianLapenna12 Yes, it's possible to support custom word lists. For example, you could take the list of words from the above site and specify the path 

```
python3 -m inclusivewriting.main en --filepath filepath
```

For example, an example custom wordlist is given here for English: https://github.com/johnsamuelwrites/inclusive/blob/master/inclusivewriting/resources/en/pronouns.json
